src/looper.cxx

```diff
@@ -195,11 +199,11 @@ void Looper::pitchShift(int count, float* input, float* output)
 		int iTemp5 = int(fTemp4);
 		float out=output[0];
 		out += (float)(((1 - fTemp3) * (((fTemp4 - iTemp5) *
-		                                 fVec0[(IOTA-int((int((1 + iTemp5)) & 65535)))&65535]) + ((0 - ((
-		                                                 fRec0[0] + fSlow3) - iTemp5)) * fVec0[(IOTA-int((iTemp5 & 65535)))
-		                                                                 &65535]))) + (fTemp3 * (((fRec0[0] - iTemp1) * fVec0[(IOTA-int((int(
-		                                                                                 iTemp2) & 65535)))&65535]) + ((iTemp2 - fRec0[0]) * fVec0[(IOTA-int((
-		                                                                                                 iTemp1 & 65535)))&65535]))));
+										 fVec0[(IOTA-int((int((1 + iTemp5)) & 65535)))&65535]) + ((0 - ((
```

Comment:
Some whitespace script seems to have reformatted this - see if you can remove this change as it isn't related to the PR.